example/server/urls.py

```diff
@@ -13,13 +13,15 @@
     2. Add a URL to urlpatterns:  path('blog', include('blog.urls'))
 """
 from django.contrib import admin
-from django.contrib.auth import views
-from django.urls import path, re_path, include
+from django.urls import path, include
+
 import polaris.urls
+from polaris.sep38 import manager_api
 from .views import all_fields_form_view, confirm_email, skip_confirm_email, log_callback
 
-
 urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("anchor/sep38/", include(manager_api)),
```

Comment:
Lets remove both of these URL paths from the reference server, they're not needed.